### PR TITLE
[HOTFIX] Ignore testcase for compatibility problem in spark 2.1

### PR DESCRIPTION
**Pull request by kevinjmh**
Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [ ] Any interfaces changed?
 
 - [ ] Any backward compatibility impacted?
 
 - [ ] Document update required?

 - [ ] Testing done
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
       
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.1/853/

File: integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/createTable/TestCreateTableLike.scala [hunk 1]
```diff
@@ -134,7 +134,9 @@ class TestCreateTableLike extends QueryTest with BeforeAndAfterEach with BeforeA
     assert(exception.getMessage.contains("already exists in database"))
   }
 
-  test("command with location") {
+  // ignore this test case since Spark 2.1 does not support specify location
+  // and also current implementation in carbon does not use this parameter.
```
[the remark this answers is not in the document]
Comment (kevinjmh) [reply integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/createTable/TestCreateTableLike.scala]: actually we ignore the location parameter in carbon. This parameter will have no effect for  spark 2.2+

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.2/859/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.2, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/861/

**Comment by akashrn5** [comment]
LGTM

**Comment by jackylk** [comment]
LGTM

**Comment by brijoobopanna** [comment]
retest this please

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.2.1, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.2/1039/

**Pull request closed without merging**